### DISCUSSION TITLE
Ask for help ! I got an error when dpgen init bulk

### DISCUSSION THREAD

**Discussion by RO-CR** [deepmodeling/dpgen#1695] [Q&A]
Hello, I met an error. The following is the output page in the run. I want to ask how to solve this problem. Thank you very much for your help.

error.log  
/home/scu401/lxx/dpgen-0.10.6/lib/python3.8/site-packages/paramiko/transport.py:219: CryptographyDeprecationWarning: Blowfish has been deprecated
  "class": algorithms.Blowfish,
/home/scu401/lxx/dpgen-0.10.6/lib/python3.8/site-packages/gromacs/__init__.py:286: GromacsImportWarning: Some Gromacs commands were NOT found; maybe source GMXRC first? The following are missing:
['release']

  warnings.warn("Some Gromacs commands were NOT found; "
Traceback (most recent call last):
  File "/home/scu401/lxx/dpgen-0.10.6/bin/dpgen", line 10, in <module>
    sys.exit(main())
  File "/home/scu401/lxx/dpgen-0.10.6/lib/python3.8/site-packages/dpgen/main.py", line 185, in main
    args.func(args)
  File "/home/scu401/lxx/dpgen-0.10.6/lib/python3.8/site-packages/dpgen/generator/run.py", line 3642, in gen_run
    run_iter (args.PARAM, args.MACHINE)
  File "/home/scu401/lxx/dpgen-0.10.6/lib/python3.8/site-packages/dpgen/generator/run.py", line 3604, in run_iter
    make_train (ii, jdata, mdata)
  File "/home/scu401/lxx/dpgen-0.10.6/lib/python3.8/site-packages/dpgen/generator/run.py", line 211, in make_train
    numb_models = jdata['numb_models']
KeyError: 'numb_models'

out.log
Version: 0.10.6
Date:    Jul-11-2022
Path:    /home/scu401/lxx/dpgen-0.10.6/lib/python3.8/site-packages/dpgen

Dependency
------------
     numpy     1.22.3   
    dpdata      0.2.7   
  pymatgen            unknown version or path
     monty  2022.4.26  
       ase     3.22.1   
  paramiko      2.8.1   
 custodian  2022.5.26   

Here are my input files

param.json

{
    "stages" : [1,2,3,4],
    "super_cell":   [1, 1, 1],
    "cell_type": "hcp",
    "form_poscar": true,
    "form_poscar_path":"/home/scu401/lxx/inputfile/bn2-23-5×3×2/POSCAR",
    "elements":     ["N","B"],
    "potcars":      ["/home/scu401/lxx/inputfile/bn2-23-5×3×2/POTCAR_N","/home/scu401/lxx/inputfile/bn2-23-5×3×2/POTCAR_B"],
    "relax_incar": "/home/scu401/lxx/inputfile/bn2-23-5×3×2/INCAR_rlx",
    "md_incar" : "/home/scu401/lxx/inputfile/bn2-23-5×3×2/INCAR_md",
    "scale":        [0.97, 1.00, 1.03],
    "skip_relax":   false,
    "pert_numb":    20,
    "md_nstep" :    25,
    "pert_box":     0.03,
    "pert_atom":    0.01,
    "coll_ndata":   5000,
    "type_map" :    ["N","B"],
    "_comment":     "that's all"
}

machine.json
{
    "api_version": "1.0",
    "train": {
        "machine": {
            "batch_type": "Shell",
            "context_type": "local",
            "remote_root": "/home/scu401/lxx/InitData_file/bn2-23-5×3×2",
            "local_root": "./"
        },
        "resources": {
            "source_list": ["/home/scu401/lxx/deepmd-kit/bin"],
            "group_size": 2,
            "number_node": 1,
            "cpu_per_node": 1,
            "gpu_per_node": 1,
            "queue_name": "newgpu",
            "kwargs": {}
        },
      "command": "dp"
    },
    "model_devi": {
        "machine": {
            "batch_type": "Shell",
            "context_type": "local",
            "remote_root": "/home/scu401/lxx/InitData_file/bn2-23-5×3×2",
            "local_root": "./"
       		},
        "resources": {
            "source_list": [
                "/opt/env/lammps-23Jun2022+deepmd_2.2.4+openmpi-4.1.5+intel2019u4+cuda-11.2"
            ],
            "batch_type": "Shell",
            "group_size": 160,
            "number_node": 1,
            "cpu_per_node": 1,
            "gpu_per_node": 1,
            "queue_name": "newgpu",
            "kwargs": {}
       			 },
        "command": "lmp-mpi"
    },
    "fp": {
        "machine": {
            "batch_type": "pbs",
            "context_type": "ssh",
            "remote_profile": {
                "hostname": "",
                "port": ,
                "password": "",
                "username": ""
           			},
            "remote_root": "/public/home/lxx/init_surface/bn2-23-5×3×2_surf",
            "local_root": "./"
        },
        "resources": {
            "batch_type": "pbs",
            "group_size": 4,
            "source_list": ["/public/software/profile.d/apps_vasp-5.4.4_intel2020_intelmpi2020.sh"],
            "number_node": 1,
            "cpu_per_node": 16,
            "queue_name": "oneNode",
            "custom_flags": [
                "#PBS -l walltime=100:00:00" 
            ],
            "kwargs": {},
            "gpu_per_node": 0
        		},
        "command": "mpirun -np 16 vasp_std"
    }
}

**Comment by njzjz** [comment]
Per the error message, it seems that you didn't run `dpgen init bulk`, but `dpgen run`.

**Reply by RO-CR** [reply]
Thanks very much for your reply. I have solved the problem.
